packages/console/src/components/AppContent/components/Sidebar/components/Section/index.module.scss

```diff
@@ -6,5 +6,9 @@
   padding: _.unit(2) _.unit(6);
   text-transform: uppercase;
   letter-spacing: 0.1em;
-  font-size: 12px;
+  font-size: 10px;
+
+  &.zh {
+    font-size: 12px;
+  }
```

Comment:
nit: is this a good practice? using language as a styling reference.

Reply:
I don't think this is necessary, as the browser itself would do this by default.

Reply:
Revert this commit, need to take a discussion.